addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -701,6 +701,15 @@ $line-height-truncate: 1.25em;
     opacity: 0;
 }
 
+.o_wslides_fs_sidebar_section {
```

Comment:
It seems to be put in a css section called "// Modals", but shouldn't it be located a bit above with lesson-specific css ? It seems there is an old section called "// New lesson page (not fullscreens)", maybe it could be added at the end of that section (aka 'not fullscreen, or common') ?

Not sure if clear.